__device-tests__/pages/editor-page.js

```diff
@@ -59,6 +59,44 @@ export default class EditorPage {
 		return elements.length > 0;
 	}
 
+	async verifyHtmlContent( html: string ) {
+		if ( Platform.OS === 'android' ) {
+			await this.verifyHtmlContentAndroid( html );
+		} else {
+			await this.verifyHtmlContentIOS( html );
+		}
+	}
+
+	async getTextViewForHtmlViewContent() {
+		const accessibilityId = 'html-view-content';
+		let blockLocator = `//*[@${ this.accessibilityIdXPathAttrib }="${ accessibilityId }"]`;
+
+		if ( ! isAndroid() ) {
+			blockLocator += '//XCUIElementTypeTextView';
+		}
+		return await this.driver.elementByXPath( blockLocator );
+	}
+
+	async verifyHtmlContentAndroid( html: string ) {
+		await toggleHtmlMode( this.driver );
+
+		const htmlContentView = await this.getTextViewForHtmlViewContent();
+		const text = await htmlContentView.text();
+		expect( text ).toBe( html );
+
+		await toggleHtmlMode( this.driver );
+	}
+
+	async verifyHtmlContentIOS( html: string ) {
```

Comment:
`verifyHtmlContentIOS` seems identical to `verifyHtmlContentAndroid` at this point. Can we consolidate the two?

Reply:
Right! done! 👍